package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "1ff-chat-ui",
-  "version": "1.0.46",
+  "version": "1.0.47",
```

Comment:
you dont need to update the version, there is a deployment script that does this for you;